packages/flutter/test/cupertino/nav_bar_test.dart

```diff
@@ -136,6 +136,70 @@ void main() {
     expect(tester.getCenter(find.text('Title')).dx, 400.0);
   });
 
+  testWidgets('Can specify custom brightness', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const CupertinoApp(
+        home: CupertinoNavigationBar(
+          backgroundColor: Color(0xF0F9F9F9),
+          brightness: Brightness.dark,
+        ),
+      ),
+    );
+
+    final AnnotatedRegion<SystemUiOverlayStyle> region1 = tester.allWidgets.singleWhere((
+        Widget widget) => widget is AnnotatedRegion) as AnnotatedRegion<SystemUiOverlayStyle>;
```

Comment:
nit: 2-space indent. I just realized that you could use `whereType` to avoid the force cast.
```suggestion
    final AnnotatedRegion<SystemUiOverlayStyle> region1 = tester.allWidgets
      .whereType<AnnotatedRegion<SystemUiOverlayStyle>>()
      .single;
```